extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -11,7 +11,8 @@ class CommunityPageSpecialUsersModel {
 	const ALL_MEMBERS_MCACHE_KEY = 'community_page_all_members';
 	const ALL_MEMBERS_COUNT_MCACHE_KEY = 'community_page_all_members_count';
 	const RECENTLY_JOINED_MCACHE_KEY = 'community_page_recently_joined';
-	const MCACHE_VERSION = '1.2';
+	const MCACHE_VERSION = '1.3';
+	const RECENTLY_JOINED_USERS_AVATAR_SIZE = 24;
```

Comment:
I know I already asked this, but is it a unique avatar size? Could we reuse an existing one / add it to AvatarService to keep it consistent?